sdk/cognitiveservices/azure-cognitiveservices-language-textanalytics/azure/cognitiveservices/language/textanalytics/_text_analytics_client.py

```diff
@@ -0,0 +1,328 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from typing import (  # pylint: disable=unused-import
+    Union,
+    Optional,
+    Any,
+    List,
+    Dict,
+    TYPE_CHECKING,
+)
+from azure.core.tracing.decorator import distributed_trace
+from ._generated.models import ErrorException
+from ._generated._text_analytics_client import TextAnalyticsClient as TextAnalytics
+from ._base_client import TextAnalyticsClientBase
+from ._response_handlers import (
+    _validate_batch_input,
+    process_batch_error,
+    entities_result,
+    linked_entities_result,
+    key_phrases_result,
+    sentiment_result,
+    language_result,
+)
+
+if TYPE_CHECKING:
+    from ._models import (
+        LanguageInput,
+        MultiLanguageInput,
+        DocumentLanguage,
+        DocumentEntities,
+        DocumentLinkedEntities,
+        DocumentKeyPhrases,
+        DocumentSentiment,
+        DocumentError,
+    )
+
+
+class TextAnalyticsClient(TextAnalyticsClientBase):
+    """The Text Analytics API is a suite of text analytics web services built with best-in-class
+    Microsoft machine learning algorithms. The API can be used to analyze unstructured text for
+    tasks such as sentiment analysis, key phrase extraction and language detection. No training data
+    is needed to use this API; just bring your text data. This API uses advanced natural language
```

Comment:
I think an em-dash would be more appropriate than a semicolon here. Absent that could use `--`